src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
+    , eventOverlapsWithAnotherEvent = "กิจกรรมนี้ซ้อนทับกับกิจกรรมอื่น"
+    , eventsCanTStartInThePast = "กิจกรรมไม่สามารถเริ่มในอดีต"
+    , failedToJoinEventThereArenTAnySpotsLeft = "ไม่สามารถเข้าร่วมกิจกรรม ไม่มีที่ว่างอีกแล้ว"
+    , failedToJoinThisEventDoesnTExistTryRefreshingThePage = "ไม่สามารถเข้าร่วม กิจกรรมนี้ไม่มีอยู่ (ลองรีเฟรชหน้าเว็บ?)"
+    , failedToLeaveEvent = "ไม่สามารถออกจากกิจกรรม"
+    , faq = "คำถามที่พบบ่อย"
+    , faq1 = "ฉันไม่ชอบที่ meetup.com เรียกเก็บเงิน ส่งอีเมลซับซ้อนถึงฉัน และดูอิ่มตัว ฉันยังอยากลองทำบางสิ่งที่มีความหมายมากขึ้นโดยใช้ "
+    , faq2 = "เพื่อดูว่าเป็นไปได้ในการใช้ที่ทำงานหรือไม่"
+    , faq3 = "ฉันเพียงแค่ใช้เงินส่วนตัวเพื่อโฮสต์มัน สิ่งนี้ไม่เป็นไรเพราะได้ออกแบบมาเพื่อให้มีค่าใช้จ่ายในการทำงานน้อยที่สุด ในกรณีที่ Meetdown ได้รับความนิยมอย่างมากและค่าใช้จ่ายในการโฮสต์เริ่มต้นมากขึ้น ฉันจะขอบริจาค"
+    , faqQuestion1 = "ใครคือผู้ที่อยู่เบื้องหลังทั้งหมดนี้?"
+    , faqQuestion2 = "ทำไมเว็บไซต์นี้ถึงถูกสร้างขึ้น?"
+    , faqQuestion3 = "ถ้าเว็บไซต์นี้ฟรีและไม่ได้วางโฆษณาหรือขายข้อมูล มันสามารถรักษาตัวเองได้อย่างไร?"
+    , forHelpingMeOutWithPartsOfTheApp = " สำหรับการช่วยฉันในส่วนของแอป"
+    , formatDate = Date.format "MMMM ddd"
+    , frequentQuestions = "คำถามที่พบบ่อย"
+    , futureEvents = "กิจกรรมในอนาคต"
+    , goToHomepage = "ไปที่หน้าหลัก"
+    , group1 = "คุณยังไม่ได้สมัครสมาชิกกับกลุ่มใด ๆ คุณสามารถทำได้โดยการกด \""
+    , groupDescription = "คำอธิบายกลุ่ม"
+    , groupName = "ชื่อกลุ่ม"
+    , groupNotFound = "ไม่พบกลุ่ม"
+    , hideU_00A0Attendees = "(ซ่อน\u{00A0}ผู้เข้าร่วม)"
+    , hoursLong = " ชั่วโมง."
+    , howManyHoursLongIsIt = "มันยาวกี่ชั่วโมง?"
+    , howManyPeopleCanJoinLeaveThisEmptyIfThereSNoLimit = "มีกี่คนที่สามารถเข้าร่วม (ปล่อยว่างถ้าไม่มีขีดจำกัด)"
+    , ifYouDontSeeTheEmailCheckYourSpamFolder = "ถ้าคุณไม่เห็นอีเมล ตรวจสอบโฟลเดอร์สแปมของคุณ"
+    , imageEditor = "ตัวแก้ไขรูปภาพ"
+    , info = "ข้อมูล"
+    , inPersonEvent = "กิจกรรมที่พบกันด้วยตัวเอง 🤝"
+    , invalidDateFormatExpectedSomethingLike_2020_01_31 = "รูปแบบวันที่ไม่ถูกต้อง คาดว่าจะมีลักษณะประมาณ 2020-01-31"
+    , invalidEmailAddress = "ที่อยู่อีเมลไม่ถูกต้อง"
+    , invalidInput = "ข้อมูลที่ป้อนไม่ถูกต้อง เขียนบางอย่างที่เป็นแบบ 1 หรือ 2.5"
+    , invalidTimeFormatExpectedSomethingLike_22_59 = "รูปแบบเวลาไม่ถูกต้อง คาดว่าจะมีลักษณะประมาณ 22:59"
+    , invalidUrlLong = "URL ไม่ถูกต้อง ใส่บางสิ่งที่เป็นแบบ https://my-hangouts.com หรือปล่อยว่าง"
+    , invalidValueChooseAnIntegerLike5Or30OrLeaveItBlank = "ค่าไม่ถูกต้อง เลือกจำนวนเต็มเช่น 5 หรือ 30 หรือปล่อยว่าง"
+    , isItI = "ฉันคือ "
+    , itsTakingPlaceAt =
+        \isPastEvent ->
+            if isPastEvent then
+                "• มันจัดขึ้นที่ "
+
+            else
+                "• มันกำลังจะจัดขึ้นที่ "
+    , iWantThisGroupToBePubliclyVisible = "ฉันต้องการให้กลุ่มนี้มองเห็นได้สาธารณะ"
+    , iWantThisGroupToBeUnlistedPeopleCanOnlyFindItIfYouLinkItToThem = "ฉันต้องการให้กลุ่มนี้ไม่ระบุ (คนอื่นจะสามารถหาได้เฉพาะเมื่อคุณส่งลิงก์ให้)"
```

Comment:
probably mistranslated

Reply:
_Key: `iWantThisGroupToBeUnlistedPeopleCanOnlyFindItIfYouLinkItToThem`_

**Original**: I want this group to be unlisted (people can only find it if you link it to them)

**Old Translation**: ฉันต้องการให้กลุ่มนี้ไม่ระบุ (คนอื่นจะสามารถหาได้เฉพาะเมื่อคุณส่งลิงก์ให้)

_Dtinth Review: Probably mistranslated_

**New Translation**: ฉันต้องการให้กลุ่มนี้ไม่ปรากฏในรายการ (คนอื่นๆจะสามารถหาเจอกลุ่มนี้ได้เฉพาะเมื่อคุณส่งลิงก์ให้กับพวกเขา)

**Why**: The new translation provides a more accurate description of the original meaning.

Reply:
```suggestion
    , iWantThisGroupToBeUnlistedPeopleCanOnlyFindItIfYouLinkItToThem = "ฉันต้องการให้กลุ่มนี้ไม่ปรากฏในรายการ (คนอื่นๆจะสามารถหาเจอกลุ่มนี้ได้เฉพาะเมื่อคุณส่งลิงก์ให้กับพวกเขา)"
```